strings/tips.txt

```diff
@@ -42,47 +42,61 @@ As the Captain, you can purchase a new emergency shuttle using a communications
+As a Medical Doctor, you can extract implants without breaking them by holding an empty implant case in your offhand while performing the extraction step.
 As a Medical Doctor, clone scanning people will implant them with a health tracker that displays their vitals in the clone records. Useful to check on crew members that didn't activate suit sensors!
 As a Chemist, there are dozens of chemicals that can heal, and even more that can cause harm. Experiment!
 As a Chemist, some chemicals can only be synthesized by heating up the contents with a chemical heater or manually with lighters and similar tools.
-As a Chemist, you will be expected to supply crew with certain chemicals. For example, clonexadone and mannitol for the cryo tubes, unstable mutagen and saltpetre for botany as well as healing pills and patches for the front desk.
+As a Chemist, you will be expected to supply crew with certain chemicals. For example, clonexadone and mannitol for the cryo tubes, plastic for crafting purposes, unstable mutagen and saltpetre for botany as well as healing pills and patches for the front desk.
 As a Chemist, you can make 100u bottles from plastic sheets. The ChemMaster can produce infinite 30u glass bottles as well and if researched the medical techfab can make beakers with capacity ranging from 120u to 300u.
 As a Chemist, you can recharge your chemical dispenser with an inducer or by replacing its cell.
 As a Chemist, Water and Potassium mixed together will create an explosion, with power scaling by amount used. Don't do it.
+As a Chemist, you can create more Bruise Packs and Burn Ointments by using the crafting menu.
+As a Chemist, you can use alternative sources for chemicals other than your chemical dispencer. For example, setting a sheet of paper on fire will create a pile of Ash which you then can scoop up with your beaker. Experiment!
 As a Geneticist, you can eject someone from cloning early by disabling power in genetics. Note that they will suffer more genetic damage and may lose vital organs from this.
 As a Geneticist, T goes to A, and G goes to C.
 As a Geneticist, becoming a hulk makes you capable of dealing high melee damage, stunlocking people, and punching through walls. However, you can't fire guns, will lose your hulk status if you go into critical condition, and are not considered a human by the AI while you are a hulk.
 As a Geneticist, you have to power to make superhumans. Be aware, going over 99 instability WILL cause side effects.
-As the Virologist, your viruses can range from healing powers so great that you can heal out of critical status, or diseases so dangerous they can kill the entire crew with airborne spontaneous combustion. Experiment!
+As a Geneticist, you can obtain chromosomes by activating mutations. You can then use these chromosomes to ehnance certain mutations.
+As a Paramedic, it is a good idea to purchase a handheld crew monitor and to keep it open at all times.
```

Comment:
afaik they start with one on their belt

Reply:
Suit slot, but yes, they do, forgot about that.